Node/src/botbuilder-teams.d.ts

```diff
@@ -213,62 +195,593 @@ export interface O365ConnectorCardActionBase {
+  
+  /** Returns the JSON for the current section */
+  toSection(): IO365ConnectorCardSection;
+}
+
+/** Card builder class that simplifies building O365 connector card section facts. */
+export declare class O365ConnectorCardFact implements IIsO365ConnectorCardFact {
+
+  /** 
+   * Creates a new O365 connector card section fact. 
+   * @param session (Optional) will be used to localize any text. 
+   */
+  constructor(session?: builder.Session);
+
+  /** Display name of the fact. */
+  name(v: string): O365ConnectorCardFact;
+
+  /** Display value for the fact. */
+  value(text: string|string[], ...args: any[]): O365ConnectorCardFact;
+
+  /** Returns the JSON for the current fact */
+  toFact(): IO365ConnectorCardFact;
+}
+
+/** Card builder class that simplifies building O365 connector card section images. */
+export declare class O365ConnectorCardImage implements IIsO365ConnectorCardImage {
+
+  /** 
+   * Creates a new O365 connector card section image. 
+   * @param session (Optional) will be used to localize any text. 
+   */  
+  constructor(session?: builder.Session);
+
+  /** URL for the image. */
+  image(url: string): O365ConnectorCardImage;
+
+  /** Alternative text for the image. */
+  title(text: string|string[], ...args: any[]): O365ConnectorCardImage;
+
+  /** Returns the JSON for the current image. */
+  toImage(): IO365ConnectorCardImage;
+}
+
+/** Card builder class that simplifies building O365 connector ViewAction action. */
+export declare class O365ConnectorCardViewAction implements IIsO365ConnectorCardActionBase {
+
+  /** 
+   * Creates a new O365 connector card ViewAction action object. 
+   * @param session (Optional) will be used to localize any text. 
+   */  
+  constructor(session?: builder.Session);
+
+  /** Name of the action that will be used as button title. */
+  name(text: string|string[], ...args: any[]): O365ConnectorCardViewAction;
+
+  /** Action Id. */
+  id(actionId: string): O365ConnectorCardViewAction;
+
+  /** Target URL. */
+  target(targetUrl: string): O365ConnectorCardViewAction;
+
+  /** Returns the JSON for the current action. */
+  toAction(): IO365ConnectorCardViewAction;
+}
+
+/** Card builder class that simplifies building O365 connector OpenUri action. */
+export declare class O365ConnectorCardOpenUri implements IIsO365ConnectorCardActionBase {
+
+  /** 
+   * Creates a new O365 connector card OpenUri action object. 
+   * @param session (Optional) will be used to localize any text. 
+   */  
+  constructor(session?: builder.Session);
+
+  /** Name of the action that will be used as button title. */  
+  name(text: string|string[], ...args: any[]): O365ConnectorCardOpenUri;
+
+  /** Action Id. */
+  id(actionId: string): O365ConnectorCardOpenUri;
+
+  /** Default target URL. It will be also used for desktop / web clients. */
+  default(targetUrl: string): O365ConnectorCardOpenUri;
```

Comment:
A targets() method that takes a map of platform -> url might feel more natural than chaining multiple calls, one for each platform.